packages/documentation/src/content/docs/integration/requirements/webhook-events.mdx

```diff
@@ -26,20 +26,28 @@ For Rafiki to notify you about webhook events, you must expose a webhook endpoin
 
 When an event occurs, the [`backend`](/integration/services/backend-service) service makes a <Badge text="POST" variant="success" size="medium"/> request to your configured webhook endpoint. The `backend` service expects a `200` status in return.
 
+<div class="overflow-table">
```

Comment:
Are these changes from elsewhere?

Reply:
I had experimented with a new Starlight component, so I had to merge main back into this branch.